template/_package.json

```diff
@@ -53,7 +66,8 @@
     "@nuxtjs/pwa": "^2.6.0"<% } %><% if (ui === 'tachyons') { %>,
     "tachyons": "^4.11.1"<% } %>
   },
-  "devDependencies": {
+  "devDependencies": { <% if (server === 'feathers') { %>
+    "@feathersjs/cli": "^3.8.5",<% } %>
```

Comment:
"nodemon" is better to be first

Reply:
Fixed in commit 9f9aa8e394dfb948e6ea533e402a9ec32e1b7534